client/js/constants.js

```diff
@@ -105,11 +106,16 @@ const timeFormats = {
 	msgWithSeconds: "HH:mm:ss",
 };
 
+const allPriviledgeModes = Object.values(modes.modes);
+allPriviledgeModes.splice(-1, 1);
```

Comment:
What's this supposed to do? Remove `normal`?

Add a comment please.

Reply:
Oh goodness, that's a good question. Yeah, I think it was to remove normal, because normal isn't a priviledged mode.